synapse/lib/storm.py

```diff
@@ -234,6 +234,10 @@ def __init__(self, snap, opts=None, user=None):
         self.runtvars = set()
         self.runtvars.update(self.vars.keys())
         self.runtvars.update(self.ctors.keys())
+        self.isModuleRunt = False
+        self.isImport = False
```

Comment:
`self.isImport` appears to be a unused class variable. is this a vestigial implementation element?

Reply:
Ah, yes. it was from one of the initial attempts. I'll remove it.